src/biokbase/narrative/contents/kbasewsmanager.py

```diff
@@ -177,11 +177,11 @@ def exists(self, path):
         return self.file_exists(path)
 
     def _wsobj_to_model(self, nar, content=True):
-        nar_id = u'ws.{}.obj.{}'.format(nar['wsid'], nar['objid'])
-        model = base_model(u'{} - {} - {}'.format(nar['saved_by'], nar_id, nar['name']), nar_id)
-        model[u'format'] = u'json'
-        model[u'last_modified'] = nar[u'save_date']
-        model[u'type'] = u'notebook'
+        nar_id = 'ws.{}.obj.{}'.format(nar['wsid'], nar['objid'])
+        model = base_model('{} - {} - {}'.format(nar['saved_by'], nar_id, nar['name']), nar_id)
```

Comment:
In future, I think we should use f-strings where possible but I doubt it's worth changing all the existing code.

Reply:
Yes! Maybe that can be a separate PR.